scripts/misc/check_landsat_orphans.py

```diff
@@ -0,0 +1,100 @@
+"""
+Read landsat gap reports for Landsat 5, Landsat 7 and Landsat 8 and check orphan scenes for cleanup
+"""
+import json
+from datetime import datetime
+from odc.aws import s3_client, s3_fetch, s3_ls, s3_dump, s3_ls_dir
+
+DEAFRICA_AWS_REGION = "af-south-1"
+DEAFRICA_LANDSAT_BUCKET_NAME = "deafrica-landsat"
+DEAFRICA_REPORT_PATH = f"s3://{DEAFRICA_LANDSAT_BUCKET_NAME}/status-report/"
+
+USGS_AWS_REGION = "us-west-2"
+USGS_S3_BUCKET_NAME = "usgs-landsat"
+
+
+def get_orphans():
+    s3 = s3_client(region_name=DEAFRICA_AWS_REGION)
+
+    print("Finding Orphans")
+    report_files = list(s3_ls_dir(uri=DEAFRICA_REPORT_PATH, s3=s3))
+
+    report_files.sort()
+
+    orphan_landsat8 = [
+        orphan_file for orphan_file in report_files if "landsat_8" in orphan_file
+    ][-1]
+    orphan_landsat7 = [
+        orphan_file for orphan_file in report_files if "landsat_7" in orphan_file
+    ][-1]
+    orphan_landsat5 = [
+        orphan_file
+        for orphan_file in report_files
+        if "landsat_5" in orphan_file and "json" in orphan_file
+    ][-1]
+
+    list_orphan_paths = []
+    for orphan in [orphan_landsat7, orphan_landsat5, orphan_landsat8]:
+        file = s3_fetch(url=orphan, s3=s3)
+        dict_file = json.loads(file.decode("utf8").replace("'", '"'))
```

Comment:
Is the `replace` required. I only checked the latest report. And I don't think this will do anything